lib/Parser/FieldOption.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\Parser;
+
+use function ltrim;
+use function str_replace;
+use function strlen;
+use function trim;
+
+final class FieldOption
+{
+    /** @var string */
+    private $name;
+
+    /** @var int */
+    private $offset;
+
+    /** @var string */
+    private $body;
+
+    /** @var int */
+    private $lineCount = 0;
+
+    public function __construct(string $name, int $offset, string $body)
+    {
+        $this->name   = str_replace(' \:', ' :', $name);
```

Comment:
>     but colons (":") inside of field names must be backslash-escaped when followed by whitespace

:thinking: it says "followed", is this really what should be done here? Or should `'\: ` be replaced with `: `?

Reply:
Haha, I finally understand the backslash thing now. Fixed it